### ISSUE TITLE
It is not working on any host other than localhost? Why?

### ISSUE THREAD

**Issue by wpraas** [waifung0207/ci_bootstrap#11]
A Database Error Occurred

Unable to connect to your database server using the provided settings.

Filename: core/Loader.php

Line Number: 347

**Comment by waifung0207** [comment]
This reason for this may due to wrong configuration in config/database.php.

For localhost, the ENVIRONMENT constant is set to "development" (please see my codes from /index.php or /backend.php), in other cases the constant will be "production" instead. This difference will lead to different configuration files being loaded by CodeIgniter (for more details you may read CodeIgniter v2 user guide [here](https://ellislab.com/codeigniter/user-guide/libraries/config.html#environments)). 

In short, for "production" environment the project will load config files from e.g. "/applications/frontend/config/production/database.php" which I guess you didn't change at this moment. 

Hope this helps.
